messybrainz/__init__.py

```diff
@@ -28,16 +28,21 @@ def submit_listens_and_sing_me_a_sweet_song(recordings):
     else:
         raise exceptions.ErrorAddingException("Failed to add data")
 
-def load_recording(mbid):
+
+def load_recording_from_msid(msid):
```

Comment:
Please add docstrings, even if the function is trivial.

Reply:
Adding docstrings for the functions in this file.